packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -111,6 +112,15 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         state.agentRegistry = IAgentRegistry(_agentRegistry);
     }
 
+    function refreshBlockVerifier()
```

Comment:
Perhaps best to also add to `IExchangeV3`?

Reply:
Okay.

Reply:
updated, please take another look.